src/components/form/examples/span-fields-schema.ts

```diff
@@ -1,6 +1,16 @@
 import { FormSchema } from '@limetech/lime-elements';
 
-export const schema: FormSchema = {
+export interface SpanFieldsFormData {
+    field1?: string;
+    field2?: number;
+    field3?: string;
+    field4?: string;
+    field5?: number;
+    field6?: string;
+    field7?: number;
+}
+
+export const schema: FormSchema<SpanFieldsFormData> = {
```

Comment:
Is the typing required now? If it's not, I'm thinking we should perhaps just leave this un-typed… it brings so little value anyway, that it would kind of bring more value in showing that it's possible to not use such strict typing… 🤷‍♂️ I dunno…

Reply:
I agree that the Generic typing don't bring that much value, even though I think it DOES bring value.
But the main issue now is that the current typing makes it really hard to work with and you need to cast a lot of stuff to get it to build. So if you don't want the generic stuff we should still fix the other stuff. i.e. sub schemas are FormSchema instead of the FormSchema | boolean.

But don't know why you want to wait? We already have merged the typing once.

Reply:
Yes, I agree. We can remove it. I fix